package.json

```diff
@@ -17,13 +13,15 @@
         "react-scripts": "4.0.3",
         "web-vitals": "^1.0.1"
     },
+    "type": "module",
     "scripts": {
         "start": "react-scripts start",
-        "build": "react-scripts build",
+        "build": "npm run build-api & react-scripts build",
         "test": "react-scripts test",
         "eject": "react-scripts eject",
         "predeploy": "npm run build",
-        "deploy": "gh-pages -d build"
+        "deploy": "gh-pages -d build",
```

Comment:
@anshul119, I'm not 100% certain about two things, but I don't know how to try them either:
- Does the `build` in `scripts.deploy` refer to `scripts.build`? This is important because this is where the JSON file is generated.
- Will GitHub allow to generate a file in the first place? I assume that it does, because all the files under _build_ are done this way. On the other hand _build_ might have different permissions.

Reply:
- Yes, it is going to run `nm run build`
- It should allow generating thee file but I don't how we can access this file

Reply:
The best way to figure this out is probably to fork the repo into my personal space and see what happens. I'll get back to you once I know more.